Makefile

```
@@ -194,6 +202,7 @@ test\:scripts:
 .PHONY: test\:coverage
 test\:coverage:
 	@$(MAKE) deps:coverage
+	$(MAKE) -C ./tests/lenses build
```

todo: Just do the above suggestion on `make deps:coverage` I made and then make a new rule like the following to call here before the `@$(MAKE) deps:coverage` line.

```
.PHONY: deps\:lens
deps\:lens:
	rustup target add wasm32-unknown-unknown
	$(MAKE) -C ./tests/lenses build
```

```suggestion
	@$(MAKE) deps:lens
	$(MAKE) deps:coverage

```

---

Yes, this is part of the plan when switching to the discussed flow